staramr/subcommand/Search.py

```diff
@@ -63,6 +63,11 @@ def _setup_args(self, arg_parser):
                                     PointfinderBlastDatabase.get_available_organisms()) + '}. Defaults to disabling search for point mutations. [None].',
                                 default=None,
                                 required=False)
+        arg_parser.add_argument('--plasmidfinder-database-type', action='store', dest='plasmidfinder_database_type', type=str,
+                                help='The bacteria type to use for plasmidfinder {' + ', '.join(
```

Comment:
Could you change **bacteria type** to **database type**?

Reply:
Fixed this in 48f2a20